app/gateway/2.7.x/configure/auth/oidc-okta.md

```diff
@@ -9,84 +9,209 @@ For information about configuring OIDC using Okta as an Identity provider
+* **auth_method**: A list of authentication methods to use with the plugin, such as passwords, introspection tokens, etc. The majority of cases use `authorization_code`, and {{site.base_gateway}} will accept all methods if no methods are specified.
+* **client_id**: The `client_id` of the OpenID Connect client registered in OpenID Connect Provider. Okta provides one to identify itself.  
+* **client_secret**: The `client_secret` of the OpenID Connect client registered in OpenID Connect Provider. These credentials should never be publicly exposed.
+* **redirect_uri**: The `redirect_uri` of the client defined with `client_id` (also used as a redirection URI for the authorization code flow).
+* **scopes**:  The scope of what OpenID Connect checks. `openid` by default; set to `email` and `profile` for this example.
+
+{% navtabs %}
+{% navtab Configure plugin with Kong Manager %}
+
+1. In Kong Manager, from the Workspaces tab, select the workspace where your
+route is configured.
+
+1. Click **Routes** in the left navigation.
+
+1. On the Routes page, select the route you have configured to protect with
+Okta and click **View**.
+
+1. Scroll to the bottom of the page and click **Add Plugin**.
+
+1. Select the **OpenID Connect** plugin.
+
+1. On the plugin's configuration, in the **Common** tab, configure the
+following, at minimum:
+
+    1. **Issuer (Discovery Document URI)**:
+
+        ```
+        https://{YOUR_OKTA_DOMAIN}/oauth2/{YOUR_AUTH_SERVER}/.well-known/openid-configuration
+        ```
+
+        The `issuer` URL can be found in your Authorization Server settings.
+
+    1. **Client ID**: `{YOUR_CLIENT_ID}`
+
+        Replace `YOUR_CLIENT_ID` with the client ID shown in your Okta
+        application's **General** page.
+
+    1. **Client Secret**: `{YOUR_CLIENT_SECRET}`
+
+        Replace `YOUR_CLIENT_SECRET` with the client secret shown in your Okta
+        application's **General** page.
+
+    1. **Auth Methods**: `Authorization code flow`
+
+        This parameter lists of all the authentication methods
+        that you want the plugin to accept. If you don't select an auth method,
+        all auth methods are allowed by default.
+
+1. Switch to the **Authorization** tab and fill out **config.scopes required** with
+`openid, email, profile`.
+
+1. Switch to the **Advanced** tab and fill out **config.redirect_uri** with
+ `https://kong.com/api`.
+
+    The `redirect_uri` should be the URI you specified earlier when configuring
+     your app.
+
+1. Click **Create** to save and apply the plugin to the Route.
+
+For a list of all available configuration parameters and what they do, see the
+[OIDC plugin reference](/hub/kong-inc/openid-connect).
+
+{% endnavtab %}
+{% navtab Configure plugin with Admin API %}
+
+With an HTTP Client, enter the following cURL command and parameters to configure the OpenID Connect plugin. Use this as a template, and modify according to your own environment and configuration values.
```

Comment:
This also feels like waaayyyyy too many words, especially "With an HTTP Client" which seems ... kinda irrelevant/at odds with cURL?